In the directory: /scratch/ejdennis/rat_BrainPipe/ClearMapCluster 
As the user: ejdennis 
on host: spock7-c0-16 
Cpus_allowed_list:	1,3,5-9,12,15-16,18-23
Array Allocation Number: 19661013
Array Index: 3
/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: /mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/Analysis/VoxelizationCode.pyx
  tree = Parsing.p_module(s, pxd, full_module_name)
['run_clearmap_cluster.py', '2', '3']
3
setting elastixlib path as /jukebox/pkgs/ELASTIX/4.8/bin, ***note this is a hack and may fail***
Elastix sucessfully initialized from path: /jukebox/pkgs/ELASTIX/4.8
Traceback (most recent call last):
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 215, in load_module
    inplace=build_inplace, language_level=language_level)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 191, in build_module
    reload_support=pyxargs.reload_support)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyxbuild.py", line 102, in pyx_to_dll
    dist.run_commands()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 186, in run
    _build_ext.build_ext.run(self)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/command/build_ext.py", line 340, in run
    self.build_extensions()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 193, in build_extensions
    ext.sources = self.cython_sources(ext.sources, ext)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 347, in cython_sources
    full_module_name=module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 778, in compile
    return compile_single(source, options, full_module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 727, in compile_single
    return run_pipeline(source, options, full_module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 515, in run_pipeline
    err, enddata = Pipeline.run_pipeline(pipeline, source)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 355, in run_pipeline
    data = run(phase, data)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 335, in run
    return phase(data)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 52, in generate_pyx_code_stage
    module_node.process_implementation(options, result)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/ModuleNode.py", line 143, in process_implementation
    self.generate_c_code(env, options, result)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/ModuleNode.py", line 411, in generate_c_code
    f = open_new_file(result.c_file)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Utils.py", line 67, in open_new_file
    os.unlink(path)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/people/ejdennis/.pyxbld/temp.linux-x86_64-3.7/pyrex/ClearMap/Analysis/VoxelizationCode.c'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "run_clearmap_cluster.py", line 107, in <module>
    from ClearMap.cluster.par_tools import resampling_operations
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/cluster/par_tools.py", line 14, in <module>
    from ClearMap.ImageProcessing.CellDetection import detectCells
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/ImageProcessing/CellDetection.py", line 38, in <module>
    import ClearMap.ImageProcessing.SpotDetection
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/ImageProcessing/SpotDetection.py", line 52, in <module>
    from ClearMap.ImageProcessing.IlluminationCorrection import correctIllumination
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/ImageProcessing/IlluminationCorrection.py", line 40, in <module>
    from ClearMap.Visualization.Plot import plotTiling
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/Visualization/Plot.py", line 24, in <module>
    import ClearMap.Analysis.Voxelization as vox
  File "/mnt/sink/scratch/ejdennis/rat_BrainPipe/ClearMapCluster/ClearMap/Analysis/Voxelization.py", line 15, in <module>
    import ClearMap.Analysis.VoxelizationCode as vox
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 462, in load_module
    language_level=self.language_level)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 231, in load_module
    raise exc.with_traceback(tb)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 215, in load_module
    inplace=build_inplace, language_level=language_level)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyximport.py", line 191, in build_module
    reload_support=pyxargs.reload_support)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/pyximport/pyxbuild.py", line 102, in pyx_to_dll
    dist.run_commands()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 186, in run
    _build_ext.build_ext.run(self)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/distutils/command/build_ext.py", line 340, in run
    self.build_extensions()
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 193, in build_extensions
    ext.sources = self.cython_sources(ext.sources, ext)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Distutils/old_build_ext.py", line 347, in cython_sources
    full_module_name=module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 778, in compile
    return compile_single(source, options, full_module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 727, in compile_single
    return run_pipeline(source, options, full_module_name)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Main.py", line 515, in run_pipeline
    err, enddata = Pipeline.run_pipeline(pipeline, source)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 355, in run_pipeline
    data = run(phase, data)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 335, in run
    return phase(data)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/Pipeline.py", line 52, in generate_pyx_code_stage
    module_node.process_implementation(options, result)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/ModuleNode.py", line 143, in process_implementation
    self.generate_c_code(env, options, result)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Compiler/ModuleNode.py", line 411, in generate_c_code
    f = open_new_file(result.c_file)
  File "/usr/people/ejdennis/.conda/envs/idisco/lib/python3.7/site-packages/Cython/Utils.py", line 67, in open_new_file
    os.unlink(path)
ImportError: Building module ClearMap.Analysis.VoxelizationCode failed: ["FileNotFoundError: [Errno 2] No such file or directory: '/usr/people/ejdennis/.pyxbld/temp.linux-x86_64-3.7/pyrex/ClearMap/Analysis/VoxelizationCode.c'\n"]
